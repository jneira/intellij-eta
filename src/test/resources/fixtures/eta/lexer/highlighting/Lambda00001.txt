ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Lambda1')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvarid ('zip'')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('zipWith')
WHITE_SPACE (' ')
IToparen ('(')
ITvarsym ('\')
ITvarid ('a')
WHITE_SPACE (' ')
ITvarid ('b')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
IToparen ('(')
ITvarid ('a')
ITcomma (',')
ITvarid ('b')
ITcparen (')')
ITcparen (')')
WHITE_SPACE ('\n')
