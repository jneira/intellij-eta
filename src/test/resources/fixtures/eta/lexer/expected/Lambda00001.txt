ITmodule ('module')
WHITE_SPACE ('')
ITconid (' Lambda')
WHITE_SPACE ('1')
ITwhere (' wher')
WHITE_SPACE ('e\n\n')
ITvocurly ('')
WHITE_SPACE ('')
ITvarid ('zip'')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITvarid ('zipWith')
WHITE_SPACE (' ')
IToparen ('(')
WHITE_SPACE ('')
ITlam ('\')
WHITE_SPACE ('')
ITvarid ('a')
WHITE_SPACE (' ')
ITvarid ('b')
WHITE_SPACE (' ')
ITrarrow ('->')
WHITE_SPACE (' ')
IToparen ('(')
WHITE_SPACE ('')
ITvarid ('a')
WHITE_SPACE ('')
ITcomma (',')
WHITE_SPACE ('')
ITvarid ('b')
WHITE_SPACE ('')
ITcparen (')')
WHITE_SPACE ('')
ITcparen (')')
WHITE_SPACE ('\n')
ITsemi ('')